### PR TITLE
Add NTP Ansible resource model

### PR DESCRIPTION
**Pull request by mingjunzhang2019**
This is to add NTP Ansible support resource model.

### CODE REVIEW

File: models/enterprise_sonic/ntp/deleted_example_03.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Using deleted
+#
+
```
Comment (kerry-meyer) [inline models/enterprise_sonic/ntp/deleted_example_03.txt:3]: Please add "before" and "after" state displays for this example. If there is no appropriate NTP "show" command to display it, please use "show running-configuration" and show just the relevant NTP part of the output.

e.g.

```
show running-configuration
                *
                *
                *
ntp authenticate
ntp authentication-key 80 md5 U2FsdGVkX1/X2M9RtUU5O6HHrLbG7RH0+nSkz4j4HQg= encrypted
ntp authentication-key 90 md5 U2FsdGVkX18PBecDvwvxBpbgG/k8xZzphnUYfQW4ktE= encrypted
                *
                *
                *

**Review by kerry-meyer — COMMENTED** [review]
The model changes look good.

I have posted one minor comment. All other files are fine with no additional changes.

File: models/enterprise_sonic/ntp/deleted_example_03.txt (hunk 1 above)
Comment (mingjunzhang2019) [reply models/enterprise_sonic/ntp/deleted_example_03.txt:10]: Added "before" and "after" states

**Review by kerry-meyer — APPROVED** [review]
The current content of this model looks fine.

Approved.

**Pull request merged**